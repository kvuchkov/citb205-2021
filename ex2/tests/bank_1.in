show AAA123
show BBB321
show CCC999
show Alan
show Bob
show Chad
wire 10000 AAA123 CCC999
show AAA123
show BBB321
show CCC999
wire 10000 Alan Chad
show AAA123
show CCC999
wire 150000 AAA123 CCC999
wire 79260 AAA123 CCC999
wire 15000 AAA123 CCC99
wire 15000 AAA12 CCC999
show AAA123
show CCC999
wire 79250 AAA123 CCC999
show AAA123
show CCC999