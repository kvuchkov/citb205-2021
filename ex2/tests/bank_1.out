Accounts loaded
AAA123 (Alan): $1000.00
BBB321 (Bob): $4000.00
CCC999 (Chad): $3000.00
AAA123 (Alan): $1000.00
BBB321 (Bob): $4000.00
CCC999 (Chad): $3000.00
$100.00 was wired successfully
AAA123 (Alan): $897.50
BBB321 (Bob): $4000.00
CCC999 (Chad): $3100.00
$100.00 was wired successfully
AAA123 (Alan): $795.00
CCC999 (Chad): $3200.00
Cannot wire amount: insufficient funds
Cannot wire amount: insufficient funds
Cannot wire amount: cannot find destination account
Cannot wire amount: cannot find source account
AAA123 (Alan): $795.00
CCC999 (Chad): $3200.00
$792.50 was wired successfully
AAA123 (Alan): $0.00
CCC999 (Chad): $3992.50
